code/controllers/subsystem/objects.dm

```diff
@@ -45,4 +52,5 @@ var/datum/subsystem/objects/SSobj
 /datum/subsystem/objects/proc/setup_template_objects(list/objects)
 	for(var/A in objects)
 		var/atom/B = A
-		B.initialize()
\ No newline at end of file
+		B.spawn_atom_to_world()
```

Comment:
I've realised it makes more sense to just call trigger_atom_spawners() as it will have the list of items needing to spawn atoms

(assuming they add themselves to the processing list appropriately of course)